### PR TITLE
Adds home bar icons

### PR DESCRIPTION
**Pull request by Stardagger**

### CODE REVIEW

File: src/UI/src/shared/layouts/header-layout.html [hunk 1]
```diff
@@ -22,16 +22,30 @@
...
+          <a class="nav-link" route-href="route: register">
+            Sign up
+            <i class="ion-compose"></i>&nbsp;
+          </a>
         </li>
         <li class="${'nav-item' + (activeRoute === 'profile' ? ' active' : '')}"
             show.bind="sharedState.currentUser && sharedState.isAuthenticated">
           <a class="nav-link" route-href="route: profile; params.bind: {name: sharedState.currentUser.username}">${sharedState.currentUser.username}</a>
         </li>
+        <<<<<<< HEAD
```
Comment (memory-thrasher) [inline src/UI/src/shared/layouts/header-layout.html]: Remove the git tags

File: src/UI/src/shared/layouts/header-layout.html [hunk 2]
```diff
@@ -22,16 +22,30 @@
...
+        =======
+        <li class="${'nav-item' + (activeRoute === 'logout' ? ' active' : '')}"
+            show.bind="sharedState.currentUser && sharedState.isAuthenticated">
+          <a class="nav-link" click.delegate="logout()"><i class="ion-log-out"></i>&nbsp;Logout</a>
+
```
Comment (memory-thrasher) [inline src/UI/src/shared/layouts/header-layout.html]: Remove blank lines, add missing li close tag

File: src/UI/src/shared/layouts/header-layout.html [hunk 3]
```diff
@@ -22,16 +22,30 @@
...
+
+          >>>>>>> 10ec65f (Adds home icons)
```
Comment (memory-thrasher) [inline src/UI/src/shared/layouts/header-layout.html]: remove git tags

**Review by memory-thrasher — CHANGES_REQUESTED** [review]

**Comment by memory-thrasher** [comment]
Drop the top commit

File: src/UI/src/shared/layouts/header-layout.html [hunk 4]
```diff
@@ -22,16 +22,30 @@
...
+            <i class="ion-compose"></i>&nbsp;
+          </a>
         </li>
         <li class="${'nav-item' + (activeRoute === 'profile' ? ' active' : '')}"
             show.bind="sharedState.currentUser && sharedState.isAuthenticated">
           <a class="nav-link" route-href="route: profile; params.bind: {name: sharedState.currentUser.username}">${sharedState.currentUser.username}</a>
         </li>
+        <<<<<<< HEAD
+        =======
+        <li class="${'nav-item' + (activeRoute === 'logout' ? ' active' : '')}"
```
Comment (memory-thrasher) [inline src/UI/src/shared/layouts/header-layout.html:40]: this li shouldn't even be here in this branch

Comment (Stardagger) [reply src/UI/src/shared/layouts/header-layout.html:40]: I'm  a  little bit confused about this one. the one on 43?

**Pull request merged**